packages/speech/src/index.js

```diff
@@ -651,12 +650,7 @@ Speech.prototype.recognize = function(file, config, callback) {
       return;
     }
 
-    var reqOpts = {
-      audio: foundFile,
-      config: config
-    };
-
-    self.request(protoOpts, reqOpts, function(err, apiResponse) {
+    self.api.Speech.syncRecognize(config, foundFile, function(err, apiResponse) {
```

Comment:
To get around this line being >80 chars, let's just rename `apiResponse` to `resp`.